### ISSUE TITLE
EM-2725 Match Manifest History Hover Text to Modal title

### ISSUE THREAD

**Issue by e-manifest** [USEPA/e-manifest#1526]
The hover text for the Manifest History Icon states History and the modal states Manifest History.  We should have the alt text match the modal it opens

!image-20221219-193706.png|width=298,height=156!

!image-20221219-193729.png|width=276,height=274! <br><br> **Acceptance Criteria** <br> (x) Alt text for Manifest History Icon states Manifest History
